sklearn/ensemble/_stacking.py

```diff
@@ -306,7 +325,7 @@ class StackingClassifier(ClassifierMixin, _BaseStacking):
         The default classifier is a
         :class:`~sklearn.linear_model.LogisticRegression`.
 
-    cv : int, cross-validation generator or an iterable, default=None
+    cv : int, cross-validation generator, iterable, or "prefit", default=None
```

Comment:
Nit: We are not consistent in the library as a whole. We can at least be consistent in this file for now:

```suggestion
    cv : int, cross-validation generator, iterable, or 'prefit', default=None
```

Same comments for `StackingRegressor`.